infra/main.bicep

```diff
@@ -355,7 +400,35 @@ module openAi 'core/ai/cognitiveservices.bicep' = {
           capacity: embeddingDeploymentCapacity
         }
       }
-    ]
+    ], useGPT4V ? [
+      {
+        name: gpt4vDeploymentName
+        model: {
+          format: 'OpenAI'
+          name: gpt4vModelName
+          version: gpt4vModelVersion
+        }
+        sku: {
+          name: 'Standard'
+          capacity: gpt4vDeploymentCapacity
+        }
+      }
+    ] : [])
+  }
+}
+
+// create computer vision for image embedding && text embedding api
+module computerVision 'core/ai/cognitiveservices.bicep' = if (useGPT4V) {
```

Comment:
`ComputerVision` is used for creating image embedding and generate embedding from query for image search